### DISCUSSION TITLE
code eror consult

### DISCUSSION THREAD

**Discussion by nandasafiqalfiansyah** [nandasafiqalfiansyah/Chatify#3] [General]
code anda eror disini tempatnya

**Comment by LowScarlet** [comment]
https://github.com/nandasafiqalfiansyah/Chatify/commit/59239aa13ae1ab3eb92a7571e79c8cafe82becf0 Fix

**Reply by nandasafiqalfiansyah** [reply]
back end proses bang  tapi masih repot bentar lagi gue chek

**Reply by nandasafiqalfiansyah** [reply]
lihat sekilas structur nya berubah jadi typescript anjir agak susah nambahi backedn nya anjir

**Reply by LowScarlet** [reply]
Pake express.js

**Reply by LowScarlet** [reply]
Pake JWT dengan Tech Postgresql x Prisma ORM

**Reply by LowScarlet** [reply]
Kita buat 2 Folder root Frontend & Backend

**Reply by nandasafiqalfiansyah** [reply]
bentar masih repot ikut bangkit saya  nanti kalau lagi free aku kabarin terserah web nya kamu apa apain silakan

**Comment by LowScarlet** [comment]
Buat Folder Root seperti ini
![image](https://github.com/nandasafiqalfiansyah/Chatify/assets/64282946/0762bbfc-5ffc-41e1-b1ab-486673c15696)

Jika sudah nanti ubah di vercel root foldernya sesuai frontend dan backend seperti ini
![image](https://github.com/nandasafiqalfiansyah/Chatify/assets/64282946/ddd689ce-4656-43bc-b041-43bcb8229daa)

Root Dir di vercelnya juga
![image](https://github.com/nandasafiqalfiansyah/Chatify/assets/64282946/f769f27d-cecc-4ecb-a85e-177cf1b23db4)
